### PR TITLE
Fix/level parser

### PR DESCRIPTION
**Pull request by txsmith**
We can now load tracks from file in comma separated format.

### CODE REVIEW

**Comment by txsmith** [comment]
There seems to be a tiny bug in the cannonball logic

File: backend/src/main/nl/tudelft/ti2806/riverrush/game/GameTrack.java [hunk 1]
```diff
@@ -22,32 +26,30 @@
 
     private static final long UPDATE_DELAY = 1000;
     public static final Integer TRACK_LENGTH = 100;
-    private static final Integer DISTANCE_INTERVAL = 5;
```
Comment (JobEngel) [inline backend/src/main/nl/tudelft/ti2806/riverrush/game/GameTrack.java]: Does removing this mean we can no longer easily set the interval of events easily?

Comment (txsmith) [inline backend/src/main/nl/tudelft/ti2806/riverrush/game/GameTrack.java]: I don't really know what you mean with an interval, as events are never repeated. We can set the y position on the track of every event as well as the x position.

File: backend/src/main/resources/simpletrack.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+8,O,8
+9,O,9
+10,R,-1
+15,O,0
+20,R,1
+25,O,0.2
+30,R,-1
+35,O,0.4
+40,R,1
+45,O,0.6
```
Comment (txsmith) [inline backend/src/main/resources/simpletrack.txt]: This line spawns an obstacle on 45% of the track progress, at 0.6 on the tracks x-axis. The last value may not be appropriate but can be changed to anything

**Comment by JobEngel** [comment]
I see nothing wrong with these changes, so this may be merged

**Pull request merged**